Xamarin.Forms.Platform.Android/StepperRendererManager.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.ComponentModel;
+using Android.Views;
+using AButton = Android.Widget.Button;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public static class StepperRendererManager
+	{
+		public static void CreateStepperButtons<TButton>(IStepperRenderer renderer, out TButton downButton, out TButton upButton)
+			where TButton : AButton
+		{
+			downButton = (TButton)renderer.CreateButton(false);
+			downButton.Id = Platform.GenerateViewId();
+			downButton.Focusable = true;
+			downButton.Text = "-";
+			downButton.Gravity = GravityFlags.Center;
+			downButton.Tag = renderer as Java.Lang.Object;
+			downButton.SetOnClickListener(StepperListener.Instance);
+
+			upButton = (TButton)renderer.CreateButton(true);
+			upButton.Id = Platform.GenerateViewId();
+			upButton.Focusable = true;
+			upButton.Text = "+";
+			upButton.Gravity = GravityFlags.Center;
+			upButton.Tag = renderer as Java.Lang.Object;
+			upButton.SetOnClickListener(StepperListener.Instance);
+
+			downButton.NextFocusForwardId = upButton.Id;
+		}
+
+		public static void UpdateButtons<TButton>(IStepperRenderer renderer, PropertyChangedEventArgs e, TButton downButton, TButton upButton)
+			where TButton : AButton
+		{
+			if (!(renderer?.Element is Stepper stepper))
+				return;
+
+			if (e == null ||
```

Comment:
The check is not your traditional null check - if null, then update, otherwise check the properties. I use this same method (and bass null explicitly) to set up the buttons after creation.